test/GAP/oscar_to_gap.jl

```diff
@@ -89,3 +89,23 @@ end
     @test convert(GAP.GapObj, x) == val
     @test GAP.GapObj(x) == val
 end
+
+@testset "GapGroup and GapGroupElem" begin
+    # `GapGroup` to GAP group, Perm
+    G = symmetric_group(5)
+    val = GAP.evalstr("SymmetricGroup(5)")
+    @test GAP.GapObj(G) == val
+    @test convert(GAP.GapObj, G) == val
+
+    # `GapGroupElem` to GAP group element, Perm
+    x = perm(G, [2,3,1,5,4])
+    val = GAP.evalstr("(1,2,3)(4,5)")
+    @test GAP.GapObj(x) == val
+    @test convert(GAP.GapObj, x) == val
+
+    # test implicit conversion
+    V = GapObj[]
+    push!(V, G) # convert GAPGroup to GapObj implicitly
+    push!(V, g) # convert GAPGroupElem to GapObj implicitly
+    @test V == [GapObj(G), GapObj(g)]
```

Comment:
In light of issue #681 by @thofma I've added the above tests, as a basis for discussion: this is what those `convert` methods give us. But is this something we *want* ?

Reply:
I fixed those lines. The `GAP.` prefix was missing.